### PR TITLE
add a stop operator to emr serverless

### PR DESCRIPTION
**Pull request by vandonr-amz**
EMR serverless currently has a `delete` operator that stops the app and deletes it, but there is no operator to just stop an app.

Since the delete operator contains a stop step, to avoid duplicating the code, I made it inherit from the new stop operator.
Usually, [composition is preferred over inheritance](https://en.wikipedia.org/wiki/Composition_over_inheritance), but in this case, since they are operators, I cannot really create an operator inside another because an operator is an object that is more complex than just the action it executes. Creating an operator inside an other operator creates new tasks in the DAG, requires a different task_id, etc.

tagging for visibility: @vincbeck @syedahsn @ferruzzi

### CODE REVIEW

File: tests/system/providers/amazon/aws/example_emr_serverless.py [hunk 1]
```diff
@@ -108,6 +109,15 @@
         job_run_id=start_job.output,
     )
     # [END howto_sensor_emr_serverless_job]
+    wait_for_job.poke_interval = 10
+
+    # [START howto_operator_emr_serverless_stop_application]
```
Comment (vincbeck) [inline tests/system/providers/amazon/aws/example_emr_serverless.py:114]: Please update doc as well and use these new tags `howto_operator_emr_serverless_stop_application` to include examples in the docs

**Review by vincbeck — CHANGES_REQUESTED** [review]

File: tests/system/providers/amazon/aws/example_emr_serverless.py (hunk 1 above)
Comment (vandonr-amz) [reply tests/system/providers/amazon/aws/example_emr_serverless.py:114]: oh yes right

File: docs/apache-airflow-providers-amazon/operators/emr/emr_serverless.rst [hunk 2]
```diff
@@ -63,6 +63,18 @@ start an EMR Serverless Job.
 
 .. _howto/operator:EmrServerlessDeleteApplicationOperator:
```
Comment (vincbeck) [inline docs/apache-airflow-providers-amazon/operators/emr/emr_serverless.rst]:
```suggestion
.. _howto/operator:EmrServerlessStopApplicationOperator:
```

File: docs/apache-airflow-providers-amazon/operators/emr/emr_serverless.rst [hunk 3]
```diff
@@ -63,6 +63,18 @@ start an EMR Serverless Job.
...
+
+You can use :class:`~airflow.providers.amazon.aws.operators.emr.EmrServerlessStopApplicationOperator` to
+stop an EMR Serverless Application.
+
+.. exampleinclude:: /../../tests/system/providers/amazon/aws/example_emr_serverless.py
+   :language: python
+   :dedent: 4
+   :start-after: [START howto_operator_emr_serverless_stop_application]
+   :end-before: [END howto_operator_emr_serverless_stop_application]
+
```
Comment (vincbeck) [inline docs/apache-airflow-providers-amazon/operators/emr/emr_serverless.rst:77]:
```suggestion

.. _howto/operator:EmrServerlessDeleteApplicationOperator:
```

**Review by vincbeck — CHANGES_REQUESTED** [review]

**Review by vincbeck — APPROVED** [review]

File: docs/apache-airflow-providers-amazon/operators/emr/emr_serverless.rst [hunk 4]
```diff
@@ -61,8 +61,21 @@ start an EMR Serverless Job.
...
+You can use :class:`~airflow.providers.amazon.aws.operators.emr.EmrServerlessStopApplicationOperator` to
+stop an EMR Serverless Application.
+
+.. exampleinclude:: /../../tests/system/providers/amazon/aws/example_emr_serverless.py
+   :language: python
+   :dedent: 4
+   :start-after: [START howto_operator_emr_serverless_stop_application]
+   :end-before: [END howto_operator_emr_serverless_stop_application]
 
+.. _howto/operator:EmrServerlessDeleteApplicationOperator:
```
Comment (vincbeck) [inline docs/apache-airflow-providers-amazon/operators/emr/emr_serverless.rst:78]:
```suggestion
.. _howto/operator:EmrServerlessDeleteApplicationOperator:

```

**Review by vincbeck — CHANGES_REQUESTED** [review]

File: docs/apache-airflow-providers-amazon/operators/emr/emr_serverless.rst (hunk 4 above)
Comment (vincbeck) [reply docs/apache-airflow-providers-amazon/operators/emr/emr_serverless.rst:78]: Build docs is failing (see tests below). I think by applying the suggested change it should fix it. In any case, you can test the doc generation locally by running the command `breeze build-docs --package-filter apache-airflow-providers-amazon`

**Review by ferruzzi — APPROVED** [review]
Nice addition, LGTM

**Review by vincbeck — APPROVED** [review]

**Review by o-nikolas — APPROVED** [review]

**Pull request merged**